### ISSUE TITLE
deliteful/Switch: on desktop, clicking it adds an unpleasant outline

### ISSUE THREAD

**Issue by AdrianVasiliu** [ibm-js/deliteful#299]
At least as it can be seen in samples/Buttons.htmln on desktop browsers clicking the Switch adds an outline to it:

![image](https://cloud.githubusercontent.com/assets/1583981/4301222/b9aeafd6-3e4c-11e4-8f59-0cb4f19fe988.png)

This is the same focus outline as when navigating via tab - due to `d-switch.d-focused` CSS rule - but I think should be avoided when simply clicking. For comparison, no such outline with dojox/mobile/Switch (while it does have the outline when tab-navigating).

Reproduced on Win7 with Chrome 37, FF 32, IE10.
